src/components/scene/stats.js

```diff
@@ -16,6 +17,9 @@ module.exports.Component = registerComponent('stats', {
 
   init: function () {
     var scene = this.el;
+
+    if (utils.getUrlParameter('stats') === 'false') { return; }
```

Comment:
We can't because this code is only run when you actually have stats attached. And there's not really a use case for needed `?stats=true`, but if we wanted to see them, we can attach stats via devtools.